builtin/providers/aws/resource_aws_ecache.go

```diff
@@ -0,0 +1,256 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/aws-sdk-go/aws"
+	elasticcache "github.com/hashicorp/aws-sdk-go/gen/elasticache"
```

Comment:
seems there is a naming inconsistency across this PR. `ecache`, `elasticcache`, `elasticache`.
Let's pick the latter.